app/forms/waste_exemptions_engine/renewal_complete_form.rb

```diff
@@ -1,21 +1,7 @@
 # frozen_string_literal: true
 
 module WasteExemptionsEngine
-  class RenewalCompleteForm < BaseForm
-    attr_accessor :reference
-
-    def initialize(registration)
-      super
-      self.reference = @transient_registration.reference
-    end
-
-    # Override BaseForm method as users shouldn't be able to submit this form
-    def submit(_params)
-      raise UnsubmittableForm
-    end
-
-    def self.can_navigate_flexibly?
-      false
-    end
+  class RenewalCompleteForm < RegistrationCompleteForm
```

Comment:
If we are trying to share some code, rather than subclassing a concrete form, I'd prefer it if we had some sort of `CompleteForm` that both of these forms used. We do have an existing pattern of using modules for this sort of thing – have a look at `AddressForm` and `DataOverviewForm`.

Reply:
I want to implement this first: https://github.com/DEFRA/ruby-services-team/issues/59 and then we can look at this. The main "issue" is that the edit competition form will not make use of it yet. We might find a different way using composition rather than inheritance after we have dealt with the issue above :)

Reply:
In that case, I'd rather have the duplication now rather than introduce subclassing one concrete form from another – let's go for consistency with the other forms and then it'll be easier to clean it all up later.